### PR TITLE
Hotfix/issue1

### PR DESCRIPTION
**Pull request by hellomhc**

### CODE REVIEW

**Comment by hellomhc** [comment]
Note

File: README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
 # progit
 
-# WE FIXED THE ISSUE 1
\ No newline at end of file
+# WE FIXED THE ISSUE 1
```
Comment (cho1072011244) [inline README.md]: Goood

File: README.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+
+# WE FIXED THE ISSUE 2
+
```
Comment (cho1072011244) [inline README.md]: Should be fixed

File: README.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+# WE FIXED THE ISSUE 3
+
+# WE FIXED THE ISSUE 4
```
Comment (cho1072011244) [inline README.md]: Fixed issued

**Pull request merged**